rock.osdeps

```diff
@@ -106,7 +106,8 @@ control/sdformat:
     ubuntu:
         '14.04,14.10,15.04,15.10': nonexistent
         '16.04,16.10,17.04,17.10': libsdformat4-dev
-        default: libsdformat6-dev
+        '18.04,20.04': libsdformat6-dev
+        default: libsdformat-dev
```

Comment:
`default` will unfortunately only work for 22.04. In 24.04 it was removed from the ubuntu packages and requires an extra osrepo (e.g., http://packages.osrfoundation.org/gazebo/ubuntu-stable)